lib/painted_rabbit/base.rb

```diff
@@ -142,13 +141,13 @@ def self.jsonify(blob)
     end
     private_class_method :jsonify
 
-    def self.current_views
-      @current_views ||= [:default]
+    def self.current_view
+      @current_view ||= views[:default]
     end
-    private_class_method :current_views
+    private_class_method :current_view
 
     def self.views
-      @views ||= { identifier: {}, default: {} }
+      @views ||= ViewCollection.new
```

Comment:
Should we change this method to `self.view_collection`?

Reply:
👍